### ISSUE TITLE
Manual restucturing ideas

### ISSUE THREAD

**Issue by peteihis** [ArtOfIllusion/ArtOfIllusion#173]
Spinning this off from the discussion in #172.

I sketched some ideas on the contents list of the hand written HTML version... May be not optimal, but basically things I'd expect to find would be:

- Art of Illusion in a nut shell (should fit on one printed page or less and that page should contain just that -- not continue to using).
- Installation (maybe an appendix or something, but adressed in the beginning)
- A crash course into using (about 4 pages sounds about right. The user should have results within 3 minutes.)
- Then the actual detailed structured manual ... like:
  - User interface and controls
  - Getting extensions
  - Object types
  - 3D Objects and their Editors
  - Textures, materials and their editors
  - Animation
  - and so on....

> what about a manual structured as a fairly lightweight tutorial, with links to various appendices for advanced/detailed information?

That might work. What I miss is that the main (manual) structure would be packed into less than ten main topics (7'd be nice). Now it seems to be expanding to much more, which does not help learning or understanding. On the other hand a manual should also have the technical side that is detailed to the last bit. --> Putting everything into one long scroll just does not work.

Also I'd like to keep each html page in reasonable length. One page for one topic. Now it just slides from one topic to next never stopping -- The reader loses the structure of thought. I don't know if it depends on the theme or if it is the Sphinx itself but Peter said "it's inherent" that one src file becomes one 1. level topic....  If that really is the case, that limits quite a lot of what can be done.

And things like quick reference pages for controls (pdf?) would be useful.

**Comment by peastman** [comment]
Personally I prefer fewer longer pages to more shorter ones.  It's way easier to scan through it looking for the information you want, or to use the browser's find command.  The sidebar gives you a list of all the subsections in the chapter and links to jump to any of them, so you still have the structure.

**Comment by Sailsman63** [comment]
>What I miss is that the main (manual) structure would be packed into less than ten main topics (7'd be nice). Now it seems to be expanding to much more,

It is, and yet it isn't. This is coming back to the 'Tutorial' vs. 'Reference' For example:

 - The 'Image Resources' Section should be thought of as an appendix. Yes, it needs to be documented. Most users, however, just need to be told that they can add an image to the scene resources by going **Scene -> Images**. The dialog itself is pretty straightforward to understand, they just need a link to the details if they get stuck.
 - Are Textures and Texture Mapping really separate top-level concepts?

>Peter said "it's inherent" that one src file becomes one 1. level topic

It's inherent that each source file becomes a *page*. We can definitely create a more nested structure if needed - just manually create mini-toctree "Tables of Contents" that are then referenced from the main one...

**Comment by peteihis** [comment]
> Personally I prefer fewer longer pages to more shorter ones.

Well, I said "reasonable". :) -- There is a point where you start to get lost in excessively long files. Let's say, I have once in a while had a small share of that too.

**Comment by peteihis** [comment]
> It's inherent that each source file becomes a page. 

Ok. Misunderstood.

**Comment by peteihis** [comment]
> Are Textures and Texture Mapping really separate top-level concepts?

No. In the original manual the topic was just divided into two files, with continuing sub topic numbers. 

![Contents4](https://user-images.githubusercontent.com/22843957/77233902-4b9b2c00-6bb3-11ea-9649-88c579a519f0.jpg)

**Comment by peteihis** [comment]
> The dialog itself is pretty straightforward to understand, they just need a link to the details if they get stuck.

Yes, your thinking would seem to support, what I was hoping for but did not quite have pictured out, keeping the main structure simple and clear but providing details to those who need them.

**Comment by peteihis** [comment]
.... they said "piecewise"....

**Comment by Sailsman63** [comment]
Length of document can also appear greater than it actually is if we don't make good use of vertical space. Your comment on #172 about grouping the mode images better is well taken.

**Comment by Sailsman63** [comment]
>.... they said "piecewise"....

??? I'm not sure what you are referencing there...

**Comment by peteihis** [comment]
> ???

Just that I _could_ have taken the time to collect all the four consequent comments into one post. :D

**Comment by Sailsman63** [comment]
I've kind of circled back to the manual, now that it looks like 3.2.0 is almost ready.

A question of content and exhaustiveness: How much exhaustive detail do we need? What parts need to be illustrated, and what can be done in short prose? Note that I'm still thinking in terms of a "Quick-Start plus appendices" approach, possibly augmented with some task-based tutorials. (How to model a *whatever*, for example)

For example, I was reviewing the basics page. This is both a "Getting Started" and a tour of *every* piece of the main window. Over 5,000 words, which is quite a chuck to digest!. Aside from separating these two functions, (And re-writing the Intro. What's there currently sounds like marketing drone to me.) do we really need:

- An extra screenshot of the main window to illustrate that it can be themed? 
- Screenshots of each of the included exporter dialogs
- Screenshots of the 'File' and 'Edit' menus
- *Exhaustive* lists of every item in the 'File' and 'Edit' menus, when most of these are standard, and behave exactly as a user should expect.
    - 'File' has *at most* 3 entries that are unique to AOI: 'Link External Object', and *possibly* 'Import' and 'Export', if only to note the default file formats, and that more can be added as plugins...
    - 'Edit' has 'Select Children' and the 'Live Duplicates' stuff...

**Comment by peastman** [comment]
Having a concise "getting started" section is a very good idea.  Once we get past that and into the main manual, I would err on the side of more detail rather than less.
